ts/src/renderer/phaser/classes/PhaserChatBubble.ts

```diff
@@ -146,6 +105,42 @@ class PhaserChatBubble extends Phaser.GameObjects.Container {
 		);*/
 	}
 
+	trimText(chatText: string): string {
+		let words = chatText;
+
+		if (words.length > 40) {
+			words = words.substring(0, 40);
+			words += '...';
+		}
+		
+		return words;
+	}
+
+	createBubble(): void {
+		const bubble = this.bubble;
+		const width = this.textObject.width * 2 + 20;
+		const height = 25;
+		const borderRadius = 5;
+
+		bubble.fillStyle(0x000000, 0.5);
+		bubble.fillRoundedRect(
+			-width / 2,
+			-height / 2,
+			width * 10 / 20,
+			height,
+			borderRadius
+		);
+
+		bubble.lineStyle(2, 0x000000, 1);
+		this.bubble.x = this.textObject.x + width / 4;
+
+		bubble.setDepth(0);
+		this.add(bubble);
```

Comment:
can remove cause we never remove the bubble. this was part of the issue with alpha stacking/faded text

Reply:
rather: move to constructor